Update Spring Boot configuration Sample

debugthings opened the pull request:
The FilterRegistrationBean referenced is depreciated and removed in older versions of Spring Boot

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

Ja-Dunn commented:
@alexandair - Hi, thanks for your contribution. We are in the process of refactoring the Azure technical articles, so we can’t merge this PR. When the refactoring is complete, we can give you a link to the new location of the article you updated so you can resubmit your update, if you want to.

ktoliver commented:
@alexandair - Thanks for your contribution to the Azure docs. On November 16th, we announced the availability of our Azure technical documentation in a new location on [docs.microsoft.com](https://docs.microsoft.com). You can [read about it on our team blog](https://docs.microsoft.com/teamblog/docs-november-update). This change means the article source lives in a new GitHub repository. Because there are some changes in the source markdown as a result of the move, we can’t pull your contribution over to the new repo without creating a ton of merge conflicts. We invite you to resubmit the change (if you are willing) to the new version of the article at [https://docs.microsoft.com/en-us/azure/virtual-machines/virtual-machines-windows-ps-create](https://docs.microsoft.com/en-us/azure/virtual-machines/virtual-machines-windows-ps-create). Thanks for your understanding.

debugthings committed:
Update Spring Boot configuration Sample

The FilterRegistrationBean referenced is depreciated and removed in older versions of Spring Boot

v-shils committed:
Update app-insights-java-get-started.md
